DEVELOPMENT.md

```diff
@@ -35,6 +35,17 @@ sudo update-grub
 sudo reboot
 ```
 
+Create a new docker profile.
```

Comment:
This has been added under the `### Testing cgroupv1` section. I think it makes sense to move it to a dedicated section like `### Exposing Docker daemon to host`.